dot/sync/tip_syncer_integeration_test.go

```diff
@@ -177,12 +173,16 @@ func TestTipSyncer_handleTick_case1(t *testing.T) {
 			targetNumber: uintPtr(fin.Number),
 			direction:    network.Descending,
 			requestData:  bootstrapRequestData,
-			pendingBlock: s.pendingBlocks.getBlock(common.Hash{0xb}),
+			pendingBlock: &pendingBlock{
+				hash:    common.Hash{0xb},
+				number:  201,
+				clearAt: time.Unix(0, 0),
+			},
 		},
 	}
-
 	w, err = s.handleTick()
 	require.NoError(t, err)
+	w[0].pendingBlock.clearAt = time.Unix(0, 0)
```

Comment:
we should assert the `clearAt` field somehow (even if it's not an exact assertion, at least verify it's set in the future or something)